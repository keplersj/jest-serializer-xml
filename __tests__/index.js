const serializer = require("..");

describe("jest-serializer-xml", () => {
  it("serializes an XML document correctly", () => {
    expect.addSnapshotSerializer(serializer);

    const testDocument = `<document>test</document>`;

    expect(testDocument).toMatchSnapshot();
  });
});
