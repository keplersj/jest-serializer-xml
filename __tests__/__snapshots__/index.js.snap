// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-serializer-xml serializes an XML document correctly 1`] = `
<document>
    test
</document>
`;
